tf_quant_finance/black_scholes/BUILD

```diff
@@ -123,17 +123,18 @@ py_test(
 
 
 py_library(
-    name = "american_option",
```

Comment:
Add this to `black_scholes` dependency 

py_library(
    name = "black_scholes",
    srcs = ["__init__.py"],
    srcs_version = "PY3",
    deps = [
        ":american_prices",
        ":implied_vol_approximation",
        ":implied_vol_lib",
        ":implied_vol_newton_root",
        ":vanilla_prices",
        # numpy dep,
        # tensorflow dep,
    ],
)